src/pat/validation/validation.js

```diff
@@ -163,6 +166,18 @@ define([
                     constraint.numericality.onlyInteger = true;
                 }
             }
+
+            // Handle fields equality
+            if (opts.equality && input.value) {
+                this.$el.find("[name=" + opts.equality + "]").each(
+                    function (idx, el) {
+                        if (input.value !== el.value) {
+                            constraint.equality = {'attribute': opts.equality, 'message': '^'+opts.message["equality"]};
```

Comment:
In my understanding of http://validatejs.org/#validators-equality the ``attribute`` attribute shouldn't be neccessary.

Reply:
Do you know what the ``'^`` is for? it looks strange, but I see all the other ``message-*`` attributes have it too.

Reply:
> In my understanding of http://validatejs.org/#validators-equality the `attribute` attribute shouldn't be neccessary.

If I remove it the tests fail. I think `attribute` is mandatory when you also specify a `message`.

> Do you know what the '^ is for? it looks strange, but I see all the other message-* attributes have it too.

No idea, I put it there to be consistent with the already existent code :)